run-tests workflow fails

First encountered in pull request https://github.com/sitcomlab/simport-learning-app/pull/140

There seems to be an issue with our dependencies: https://github.com/sitcomlab/simport-learning-app/runs/5112595100?check_suite_focus=true

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: simport-learning@0.0.1
npm ERR! Found: @angular/common@12.0.5
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~12.0.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@">=13.0.0" from ng-apexcharts@1.7.0
npm ERR! node_modules/ng-apexcharts
npm ERR!   ng-apexcharts@"^1.5.12" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/runner/.npm/eresolve-report.txt for a full report.
```

--- a/.github/workflows/build-android.yml
+++ b/.github/workflows/build-android.yml
@@ -30,7 +30,7 @@ jobs:
 
       - name: Run ionic build
         run: |
-          npm install npm -g
+          npm install npm@7 -g
           npm config set package-lock false
           npm install --silent
           ionic cap build android --no-interactive --no-open --confirm --prod

--- a/.github/workflows/build-ios.yml
+++ b/.github/workflows/build-ios.yml
@@ -30,7 +30,7 @@ jobs:
 
       - name: Run ionic build
         run: |
-          npm install npm -g
+          npm install npm@7 -g
           npm config set package-lock false
           npm install --silent
           ionic cap build ios --no-interactive --no-open --confirm --prod

--- a/.github/workflows/distribute-android.yml
+++ b/.github/workflows/distribute-android.yml
@@ -30,7 +30,7 @@ jobs:
 
       - name: Run ionic build
         run: |
-          npm install npm -g
+          npm install npm@7 -g
           npm config set package-lock false
           npm install --silent
           ionic cap build android --no-interactive --no-open --confirm --prod

--- a/.github/workflows/distribute-ios.yml
+++ b/.github/workflows/distribute-ios.yml
@@ -30,7 +30,7 @@ jobs:
 
       - name: Run ionic build
         run: |
-          npm install npm -g
+          npm install npm@7 -g
           npm config set package-lock false
           npm install --silent
           ionic cap build ios --no-interactive --no-open --confirm --prod

--- a/.github/workflows/run-tests.yml
+++ b/.github/workflows/run-tests.yml
@@ -29,7 +29,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          npm install npm -g
+          npm install npm@7 -g
           npm config set package-lock false
           npm install
 

--- a/.github/workflows/run-tests.yml
+++ b/.github/workflows/run-tests.yml
@@ -31,7 +31,7 @@ jobs:
         run: |
           npm install npm -g
           npm config set package-lock false
-          npm install --silent
+          npm install
 
       - name: Run tests
         run: npm test -- --no-watch --no-progress --browsers=ChromeHeadlessCI

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "haversine-distance": "^1.2.1",
     "leaflet": "^1.7.1",
     "moment": "^2.29.0",
-    "ng-apexcharts": "^1.5.12",
+    "ng-apexcharts": "~1.5.12",
     "rxjs": "~6.5.5",
     "tslib": "^2.3.0",
     "zone.js": "~0.11.4"

